### PR TITLE
Hw4 task3 - add my_precious_logger function implementation and tests

### PR DESCRIPTION
**Pull request by IuliiaLanchuk**

### CODE REVIEW

File: homework4/task3/tests/test_get_print_output.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from project3__get_print_output.task3_get_print_output import my_precious_logger
+
+
+def test_my_precious_logger_positive(capsys):
+    my_precious_logger("OK")
+    result = capsys.readouterr()
+
+    assert result.out == "OK"
```
Comment (Cjkjvfnby) [inline homework4/task3/tests/test_get_print_output.py]: Should we test something else?

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]
Let's add at least one more test case.

File: homework4/task3/tests/test_get_print_output.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        ("OK", "OK"),
+        ("", ""),
+        ("ERROR", "ERROR"),
+    ],
+)
+def test_my_precious_logger_positive(capsys, string, expected_result):
+    my_precious_logger(string)
+    result = capsys.readouterr()
+
+    assert result.out == expected_result
```
Comment (Cjkjvfnby) [inline homework4/task3/tests/test_get_print_output.py:17]: Which side effects you function does?  Do we test them all?

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]

File: homework4/task3/tests/test_get_print_output.py (hunk 2 above)
Comment (IuliiaLanchuk) [reply homework4/task3/tests/test_get_print_output.py:17]: You mean to assert both result.out and result.err? Fixed

**Review by Cjkjvfnby — APPROVED** [review]

**Pull request merged**